swagger_server/controllers/topology_controller.py

```diff
@@ -27,11 +27,10 @@
 db_instance.initialize_db()
 
 
-def find_between(s, first, last):
+def find_after(s, delimiter):
     try:
-        start = s.index(first) + len(first)
-        end = s.index(last, start)
-        return s[start:end]
+        start = s.index(delimiter) + len(delimiter)
+        return s[start:]
     except ValueError:
         return ""
```

Comment:
How about something like this:

```python
def find_domain_name(topology_id: str) -> str:
    """
    Find domain name from topology id.

    Topology IDs are expected to be of the format
    "urn:ogf:network:sdx:topology:zaoxi.net"
    """
    *_, domain_name = topology_id.split(":")
    return domain_name
```

I find `find_after()` harder to read because the logic is slightly more complicated and the method name is too general to be immediately apparent.

Reply:
Sounds good. I passed in a delimiter, trying to make this a bit more generic.